provider/datagen/Cargo.toml

```diff
@@ -58,7 +58,7 @@ icu_unitsconversion = { workspace = true, features = ["datagen"], optional = tru
 icu_codepointtrie_builder = { workspace = true }
 icu_collections = { workspace = true, features = ["serde"] }
 icu_locid = { workspace = true, features = ["std", "serde"] }
-icu_provider = { workspace = true, features = ["std", "logging", "datagen"]}
+icu_provider = { workspace = true, features = ["std", "logging", "datagen", "experimental"]}
```

Comment:
Okay, it's datagen. I didn't want the experimental feature globally enabled for library code but I guess it's fine here, especially since it gets transitively enabled from icu_transliterate.